config/config.go

```diff
@@ -166,6 +169,13 @@ func (s *HTTPProbe) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err := unmarshal((*plain)(s)); err != nil {
 		return err
 	}
+	// The UnmarsharlYAML method of HTTPClientConfig isn't called explicitly
+	// because it isn't a pointer. and it can't be made a pointer otherwise the
```

Comment:
It is a pointer.

Reply:
You should clarify what "it" is

Reply:
What about `s.HTTPClientconfig isn't a pointer so its its UnmarsharlYAML method isn't called by the parser. And it can't be made a pointer otherwise ...`?

Reply:
It can be made a pointer, so I'd remove that bit.

Reply:
Yeah even simpler 🙂